core/src/test/java/io/aiven/klaw/service/UsersTeamsControllerServiceTest.java

```diff
@@ -284,6 +284,7 @@ void deleteUserFailureHasRequests() throws KlawException {
     when(mailService.getUserName(any())).thenReturn("testuser");
     when(manageDatabase.getRolesPermissionsPerTenant(anyInt())).thenReturn(new HashMap<>());
     when(handleDbRequests.getAllComponentsCountForUser("testuser", 101)).thenReturn(1);
```

Comment:
I think this (line 286) is no longer being used as its being replaced by new line 287.

Reply:
makes sense
thanks